hack/tools.go

```diff
@@ -1,3 +1,4 @@
+//go:build tools
```

Comment:
Is this change still needed?

Reply:
Yes, that's why I pushed 3 times :-)

CI was complaining about it

Reply:
wondering if we can remove the following line`+build tools`.

Reply:
I think we can't. CI compares the generated file with what's in this PR and complains if they are different.

Reply:
```
diff --git a/hack/tools.go b/hack/tools.go
index 7a21725c..d1427a67 100644
--- a/hack/tools.go
+++ b/hack/tools.go
@@ -1,4 +1,4 @@
-// +build tools
+//go:build tools
```

This is if I remove the other line